corsheaders/middleware.py

```diff
@@ -107,16 +106,13 @@ def process_response(self, request, response):
         if not enabled:
             return response
 
-        # todo: check hostname from db instead
-        url = urlparse(origin)
```

Comment:
This should be restored so we only `urlparse` once, and pass the `url` down into the two functions that use it, we don't want to do the work twice :)